third_party/ibis/ibis_mssql/__init__.py

```diff
@@ -109,3 +114,74 @@ def list_primary_key_columns(self, database: str, table: str) -> list:
         with self.begin() as con:
             result = con.exec_driver_sql(list_pk_col_sql, parameters=(database, table))
             return [_[0] for _ in result.cursor.fetchall()]
+
+    LIST_DATABASE_SQL = """
+        SELECT schema_name FROM information_schema.schemata
+        WHERE schema_name LIKE '%{schema_like}%'
```

Comment:
If possible I think we should use `?` bind variables for SQL parameters, like in `list_primary_key_columns`

This protects against SQL injection.